BraveWalletTests/SwapTokenStoreTests.swift

```diff
@@ -11,12 +11,12 @@ class SendSwapStoreTests: XCTestCase {
     
     func testDefaultSellBuyTokensOnMainnetWithoutPrefilledToken() {
         let store = SwapTokenStore(
-            keyringController: TestKeyringController(),
-            tokenRegistry: TestTokenRegistry(),
-            rpcController: TestEthJsonRpcController(),
-            assetRatioController: TestAssetRatioController(),
-            swapController: TestSwapController(),
-            transactionController: TestEthTxController(),
+            keyringService: MockKeyringService(),
+            blockchainRegistry: MockBlockchainRegistry(),
+            rpcService: MockJsonRpcService(),
+            assetRatioService: MockAssetRatioService(),
+            swapService: MockSwapService(),
+            txService: MockEthTxService(),
```

Comment:
Should eventually swap out these with new Test* classes (not needed for this PR, but maybe can open an issue to do later)

Reply:
created #4930